themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -14,32 +14,37 @@ meta_image: /images/challenge/challenge_cta.png
   <div class="w-full md:w-1/2">
     <h3>Startup in a Box</h3>
     <p class="pr-12">
-      Thinking about turning that side project into a little something more? Follow along to stand up a website for your startup on Amazon S3 with Cloudfront and Checkly, all using Pulumi. When you're done, we'll send you a fancy drink tumbler with a special Pulumipus on it, just for this Challenge!
+      Thinking about turning that side project into a little something more? Follow along to stand up a website for your startup on your favourite cloud provider (AWS or Google Cloud) and Checkly, all using Pulumi. When you're done, we'll send you a fancy drink tumbler with a special Pulumipus on it, just for this Challenge!
     </p>
-    <h3>Prerequisites</h3>
-    <p>In order to complete this challenge, you'll need a couple things set up in advance.</p>
-    <ul>
-      <li>
-        A <a href="https://app.pulumi.com/signup" target="_blank" rel="noopener noreferrer">Pulumi account</a>
-      </li>
-      <li>
-        The <a href="{{< relref "/docs/get-started/install/" >}}" target="_blank" rel="noopener noreferrer">Pulumi CLI</a>
-      </li>
-      <li>
-        AWS account
-      </li>
-      <li>
-        <a href="https://www.checklyhq.com/" target="_blank" rel="noopener noreferrer">Checkly</a> account
-      </li>
-    </ul>
   </div>
-
   <div class="w-full order-first md:order-last md:w-1/2">
     <img class="hidden md:block" src="/images/pulumi-challenge-swag-tumbler.png" />
     <img class="mx-auto my-4 md:hidden" src="/images/pulumi-challenge-swag-tumbler-mobile.png" />
   </div>
 </div>
 
+{{< chooser cloud "aws,gcp" >}}
```

Comment:
just to make this simpler, i wouldn't embed this into a chooser. i would just have bullet three say "AWS or Google Cloud account"

Reply:
this chooser will let a user change the whole view, text and code samples between AWS and GCP. 

We are not very familiar with your Hugo website structure guidelines - feel free change accordingly if you want :)

Reply:
Given the differences in code and terminology throughout the article, using a chooser is probably the best approach. However, we'll probably want to recompose the article a bit to streamline it. I'll get in touch with you regarding some suggested changes.